python/sglang/srt/layers/layernorm.py

```diff
@@ -143,4 +170,7 @@ def extra_repr(self):
     logger.info(
         "sgl-kernel is not available on Non-NV platforms. Fallback to other kernel libraries."
```

Comment:
Can you fix the comment - `sgl-kernel` also exists on ROCm, also simply the cascaded `if`s to:
```
if _is_hip:
   <...>
elif not _is_cuda:
   logger.info("Fallback to other kernel libraries.")
   <...>
```